features/mirror_spec.rb

```diff
@@ -18,6 +19,8 @@
   let(:metadata_files) { %w[repomd.xml repomd.xml.asc repomd.xml.key *-primary.xml.gz *-filelists.xml.gz *-other.xml.gz] }
 
   it 'has valid metadata mirrored' do
+    expect(File.join(path, 'repodata/repomd.xml')).to exist
```

Comment:
```suggestion
    expect(File.exist?(File.join(path, 'repodata/repomd.xml'))).to eq(true)
```
Sadly the initial attempt does not work correctly. Falling back to exist(path) == true works